.github/workflows/pr-python.yaml

```diff
@@ -47,3 +47,34 @@ jobs:
           make gen
           make test
 
```

Comment:
We should add the path `proto/**` to `on.pull_request.paths.`

Reply:
[d931295](https://github.com/bucketeer-io/bucketeer/pull/414/commits/d93129573ae4d9f1a10ddccc8eff56d45e38b81d)